atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitions.java

```diff
@@ -213,6 +203,18 @@ static boolean isMatchingCf(CfDef clientSide, CfDef clusterSide) {
                     clusterSide.isSetPopulate_io_cache_on_flush());
             return false;
         }
+        if (clientSide.min_index_interval != clusterSide.min_index_interval) {
```

Comment:
do you not want to return false in both of these cases?

Reply:
yep, I do - good spot!

Reply:
(ah, and the tests didn't catch these because they found _the first difference_ between the dense/wide-row tables and the other ones)